cli/daemon/dash/dashapp/src/components/api/cm/CM.tsx

```
@@ -67,14 +67,15 @@ export default class CM extends React.Component<Props> {
   }
 
   render() {
+    const shadow = this.props.noShadow ? "" : "shadow-inner "
+
     return (
       <div ref={this.container}
-          className={`relative h-full font-mono subpixel-antialiased shadow-inner ${this.props.className}`}>
+          className={`relative h-full font-mono subpixel-antialiased ${shadow}${this.props.className}`}>
```

Nit: you could keep the space between `${shadow} and ${this.props.className}`; it's harmless and makes the code a bit more resilient in case someone refactors the `shadow` definition and forgets the trailing space (CSS fails silently so it's easy to miss).